python/cudf/cudf/core/_compat.py

```diff
@@ -9,7 +9,6 @@
 PANDAS_GE_201 = PANDAS_VERSION >= version.parse("2.0.1")
 PANDAS_GE_210 = PANDAS_VERSION >= version.parse("2.1.0")
 PANDAS_GE_214 = PANDAS_VERSION >= version.parse("2.1.4")
-PANDAS_GE_220 = PANDAS_VERSION >= version.parse("2.2.0")
 PANDAS_LT_203 = PANDAS_VERSION < version.parse("2.0.3")
 PANDAS_GE_220 = PANDAS_VERSION >= version.parse("2.2.0")
```

Comment:
Can we sort these, so we have all the `EQ` then `GE` then `LT`? It looks like you deleted a duplicate line but it ended up breaking up the groupings.

Reply:
Yup, I'll do that in a follow-up pr. Potentially there is still some cleanup to be done.